agentops/client.py

```diff
@@ -268,7 +268,7 @@ def end_session(self,
         if token_cost == 'unknown':
             print('🖇 AgentOps: Could not determine cost of run.')
         else:
-            print('🖇 AgentOps: This run cost ${:.6f}'.format(float(token_cost)))
+            print('🖇 AgentOps: This run cost ${}'.format('{:.2f}'.format(token_cost) if token_cost == 0 else '{:.6f}'.format(float(token_cost))))
```

Comment:
Why cast to float?

Also, as a general rule, it's advised to use the `Decimal` module. Decimal helps avoid float precision errors when going between types.

Reply:
Sure thing can use Decimal -- used float because that was what was originally implemented in main branch